### ISSUE TITLE
mkvirtualenv

### ISSUE THREAD

**Issue by gblinick** [Satalia/production-data-science#22]
Hello,
I know this is a basic question, but I'm not sure how to get out of the gate using mkvirtualenv to set up the virtual environment. I get the response "bash: mkvirtualenv: command not found".
I am using a debian instance on GCP, if that helps. 
Any help is much appreciated.

**Comment by FilippoBovo** [comment]
Hi @gblinick,

Did you install `virtualenvwrapper` following this guide, https://virtualenvwrapper.readthedocs.io/en/latest/install.html?

**Comment by gblinick** [comment]
Hi @FilippoBovo ,

Thank you very much for the prompt reply. I was able to get it to work in the end (just now actually).
There were a few issues along the way:
1) I was using pip3 instead of pip, and I think that was causing problems.
2) After adding the 3 lines in the docs (https://virtualenvwrapper.readthedocs.io/en/latest/install.html), I had to change the 3rd line by finding where my virtualenvwrapper.sh actually was because it wasn't in /usr/local/bin/virtualenvwrapper.sh. I did this by using "find / -name virtualenvwrapper.sh". It ended up being in /home/gideon_blinick/.local/bin/virtualenvwrapper.sh. I guess this is where it is for every default GCP instance.
3) Even after changing the 3rd line, I was getting the error "ERROR: virtualenvwrapper could not find virtualenv in your path". I solved this by following the instructions here (https://stackoverflow.com/questions/31274642/error-virtualenvwrapper-could-not-find-virtualenv-in-your-path) and reinstalling virtualenv even though it was already in my system.

I don't understand that much how all the errors work but am happy that I could fix them. If I was a little descriptive it was so that other people could see my steps if they have the same problem. Again, thanks for the quick reply. I had seen the docs last night, but it was good to come back to them and the problem with a fresh mind.
